src/app/ApplicationTemplate.UWP/Views/Content/Authentication/CreateAccountPage.xaml

```diff
@@ -24,30 +24,20 @@
 
 	<Grid Background="{StaticResource MaterialOnBackgroundBrush}">
 
-		<Grid.RowDefinitions>
-			<RowDefinition Height="Auto" />
-			<RowDefinition Height="*" />
-		</Grid.RowDefinitions>
-
 		<!-- Background -->
 		<Image Source="ms-appx:///Assets/Background-Mustaches.png"
 			   Stretch="UniformToFill"
-			   Grid.RowSpan="2"
 			   HorizontalAlignment="Stretch"
 			   VerticalAlignment="Stretch" />
 
-		<!-- Back button -->
-		<Button Style="{StaticResource LeftArrowButtonStyle}"
-				Command="{Binding NavigateBack}"
-				Margin="16,56,16,0"
-				HorizontalAlignment="Left" />
-
-		<ScrollViewer Grid.Row="1">
-			<StackPanel VerticalAlignment="Center">
+		<ScrollViewer toolkit:VisibleBoundsPadding.PaddingMask="Top,Bottom">
```

Comment:
Make sure content is not cropped when scrolling.
When scrolling content on a device with rounded corners, the content should go through the rounded corners area and have enough padding to sit outside the rounded corners area.